src/composables/secureStorageRef.ts

```diff
@@ -128,10 +127,10 @@ export function useSecureStorageRef<T = string | object | any[]>(
       options.onRestored?.(val);
       // Do not write null values to the mnemonic storage
       if (val || (storageKey !== STORAGE_KEYS.mnemonic)) {
-        innerState.value = await getEncryptedValue(val);
+        encryptedState.value = await getEncryptedValue(val);
       }
     }, { immediate: true, deep: true });
   })();
 
-  return decryptedState;
+  return { encryptedState, decryptedState };
```

Comment:
I'm wondering if maybe returning those 2 values as an array would be a better approach. React is doing this. This way when using this composable instead of:
```
const {
  encryptedState: something,
  decryptedState: anotherThing,
} = ...
```

You would go with:
```
const [something, anotherThing] = ...
```